BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -247,6 +247,12 @@
   gPlatformCommonLibTokenSpaceGuid.PcdUsbKeyboardPollingTimeout | $(USB_KB_POLLING_TIMEOUT)
   gPlatformCommonLibTokenSpaceGuid.PcdLowestSupportedFwVer		| $(LOWEST_SUPPORTED_FW_VER)
 
+  !if ($(BOARD_PKG_NAME) == TigerlakeBoardPkg) || ($(BOARD_PKG_NAME) == ElkhartlakeBoardPkg)
```

Comment:
This should not be done in core.   Instead, define a MACRO to init the PCD and let BoardConfig.py to init the MACRO for specific platform.

Reply:
Sure. I would add this param as part of BoardConfig.